packages/linter/src/executors/eslint/lint.impl.ts

```diff
@@ -39,6 +39,9 @@ export default async function run(
     ? join(options.cacheLocation, projectName)
     : undefined;
 
+  const printConfigForFile = options.printConfig;
+  delete options.printConfig;
+
```

Comment:
Remove this object mutating. We can use `options.printConfig` directly in the condition below.

```suggestion
```